### ISSUE TITLE
Get log-files from docker-container to host

### ISSUE THREAD

**Issue by Z3roCoder** [sameersbn/docker-gitlab#347]
Hello there. I currently have a pretty huge problem I have no idea how to solve.

So as of right now, I am trying to get the logs contained in /home/git/gitlab/log to be on my host-system as well. I have tried configuring the docker-compose.yml the following:

under "volumes" I have added:
- /srv/docker/gitlab/gitlab/log:/home/git/gitlab/log

And upon start and doing some stuff like adding Projects, log files are in the docker container but not in my host system. This should work according to docker documentation, or am I doing something wrong?

Also I have completely removed gitlab, modified docker-compose file and then re-created everything from scratch and it still didn't work. 

Would someone know what I am doing wrong here?

**Comment by sameersbn** [comment]
@Z3roCoder that is wrong. all logs are collected at `/var/log/gitlab`. You should mount a volume at that location: `/srv/docker/gitlab/gitlab/log:/var/log/gitlab`

**Comment by Z3roCoder** [comment]
Why are then still logs located at /home/git/gitlab/log ? Even in config-files that is the log-location they are beeing written into

**Comment by sameersbn** [comment]
@Z3roCoder those are symlinks to the logs in `/var/log/gitlab`
